source/administration/command-line-tools.rst

```diff
@@ -587,33 +644,12 @@ mattermost config migrate
     .. code-block:: none
 
        ./mattermost config migrate  path/to/config.json "postgres://mmuser:mostest@dockerhost:5432/mattermost_test?sslmode=disable&connect_timeout=10"
-       
-mattermost config reset
```

Comment:
This would also need to be removed from the table of contents https://github.com/mattermost/docs/blob/e2b8f3b558288663b925d7ed3fa8970191c4f134/source/administration/command-line-tools.rst#mattermost-config.

Reply:
I've re-added that command to the CLI docs - I must have deleted it inadvertently when I was trying to refine the formatting or copying content across. Thanks!